data/pages-content/sponsor/information.md

```diff
@@ -10,6 +10,9 @@ subtitle:
 
 
 ### Due Dates of Deliverables
+<h5> Variable Dates</h5>
+- [Parcel Delivery for Booth Items](#parcel): (all exhibitors wishing to ship exhibit items)
```

Comment:
Maybe "wishing" -> "who want", it would look like this: `(all exhibitors who want to ship their exhibit items)`

Reply:
I would use the same title here and on line 199. It's a bit confusing to click on "Parcel Delivery for Booth Items" and after click on "Parcel Delivery & Gifts" to see the details.

Reply:
I've now changed it to Parcel Delivery & Swag for both.